changelogs/head.asciidoc

```diff
@@ -11,7 +11,8 @@ https://github.com/elastic/apm-server/compare/7.9\...master[View commits]
 
 * Transaction metrics aggregation now flushes on shutdown, respecting apm-server.shutdown_timeout {pull}3971[3971]
 * De-dot Jaeger process tag keys, fixing indexing errors when using jaeger-php {pull}4191[4191]
-* Fix json schema validation on `metadata.service.*` fields {pull}[4142]4142
+* Fix json schema validation on `metadata.service.*` fields {pull}4142[4142]
+* Fix a panic when apm-server.sampling.keep_unsampled is false {pull}4215[4215]
```

Comment:
Is this bug out there already? If not I don't think we need an entry here

Reply:
Technically it is in 7.9, but it's undocumented so I'll remove it. I think we had this discussion before :)